etc/contracts-test-data/contracts/loadnext/query_write_metrics.sql

```diff
@@ -0,0 +1,58 @@
+-- calculate distribution of initial and repeated writes per transaction
+
+\set start_from_miniblock_number 40000000;
+\set miniblock_range 10000;
+
+select
+    -- initial writes
+    stddev_samp(initial_writes_per_tx)                                   as initial_writes_stddev,
+    avg(initial_writes_per_tx)                                           as initial_writes_avg,
+    percentile_cont(0.00) within group (order by initial_writes_per_tx)  as initial_writes_pct_00,
+    percentile_cont(0.01) within group (order by initial_writes_per_tx)  as initial_writes_pct_01,
+    percentile_cont(0.05) within group (order by initial_writes_per_tx)  as initial_writes_pct_05,
+    percentile_cont(0.25) within group (order by initial_writes_per_tx)  as initial_writes_pct_25,
+    percentile_cont(0.50) within group (order by initial_writes_per_tx)  as initial_writes_pct_50,
+    percentile_cont(0.75) within group (order by initial_writes_per_tx)  as initial_writes_pct_75,
+    percentile_cont(0.95) within group (order by initial_writes_per_tx)  as initial_writes_pct_95,
+    percentile_cont(0.99) within group (order by initial_writes_per_tx)  as initial_writes_pct_99,
+    percentile_cont(1.00) within group (order by initial_writes_per_tx)  as initial_writes_pct_100,
```

Comment:
Does anyone know how to make this kind of thing shorter, while still keeping all of the percentiles in separate columns?